src/firefly/js/visualize/Histogram.jsx

```diff
@@ -7,13 +7,15 @@ import ReactHighcharts from 'react-highcharts/bundle/highcharts';
 import numeral from 'numeral';
 import {getFormatString} from '../util/MathUtil.js';
 
+
 var Histogram = React.createClass(
```

Comment:
Yes, I have created a DM-6287 for refactoring. I will add converting to es6 classes to it.